requirements.txt

```
@@ -1,4 +1,4 @@
-allure-python-commons
+allure-python-commons<=2.13.0
```

```suggestion
allure-python-commons<2.13.0
```
`<=` includes 2.13.0, which is broken